assets/js/app/battles/components/ListBattle/ListBattle.js

```diff
@@ -6,7 +6,7 @@ import PropTypes from 'prop-types';
 import axios from 'axios';
 import styled from 'styled-components';
 import { isEmpty } from 'lodash';
-import actions from '../../../../actions';
+import actions from 'actions';
 import Urls from '../../../../utils/urls';
```

Comment:
You can do the same thing here for the `utils` directory.

Reply:
Sorry, done.